app/src/main/java/com/keylesspalace/tusky/util/NumberUtils.kt

```diff
@@ -8,7 +8,6 @@ import kotlin.math.abs
 import kotlin.math.ln
 import kotlin.math.pow
 
-private val numberFormatter: NumberFormat = NumberFormat.getInstance()
```

Comment:
The static `NumberFormat` is bad because it doesn't update when the locale changes.